prottp.go

```diff
@@ -37,10 +40,22 @@ func Wrap(service Service) http.Handler {
 	return mux
 }
 
+var marshaler = jsonpb.Marshaler{
+	EnumsAsInts:  false,
+	EmitDefaults: false,
+	Indent:       "\t",
+	OrigName:     true,
```

Comment:
Godoc isn't so helpful here:

```
    // Whether to use the original (.proto) name for fields.
    OrigName bool
```

Looking at the [Protocol Buffer Style Guide](https://developers.google.com/protocol-buffers/docs/style#message-and-field-names):

> Use CamelCase (with an initial capital) for message names – for example, SongServerRequest. Use underscore_separated_names for field names – for example, song_name.

So the JSON will use `underscore_separated_names` I guess.